evm/15_core-concepts/46_EOAs-vs-Smart-Contracts.md

```diff
@@ -0,0 +1,85 @@
+---
+title: EOAs vs Smart Contracts
+contributors:
+  - { name: Esteban Saá, github: stebansaa }
+---
+  
+## EOAs vs Smart Contracts
+
+### What are EOAs?
+EOA stands for Externally Owned Account. In the context of EOSEVM, an EOA is an account that represents an individual user or entity on the EOSEVM network. It is similar to the concept of an Ethereum account. EOAs have their own addresses, private keys, and balances.
+
+##### What can you do with an EOA?
+With an EOA, you can perform various actions on the EOSEVM network, including:
+
+- Sending and receiving EOSEVM tokens or assets.
+- Interacting with smart contracts by calling their functions.
+- Participating in transactions and signing them with your private key.
+- Managing your own account balance and transactions.
+
+#### What you cant do with an EOA?
+There are certain limitations to what you can do with an EOA:
+
+- An EOA cannot directly execute code or perform complex computations.
+- EOAs cannot store data or maintain state on the blockchain.
+- They cannot create new smart contracts on their own.
+
+#### Wallets
+To interact with an EOA, users typically utilize wallets. Wallets provide a user-friendly interface for managing private keys, sending transactions, and interacting with the EOSEVM network. Wallets can be desktop applications, mobile apps, or web-based interfaces.
+
+#### EOAs on EOSEVM
```

Comment:
Let's remove this part, as there's really no difference between an EOA on EOS EVM or any other EVM. No need to poke the bear.

Reply:
ok.